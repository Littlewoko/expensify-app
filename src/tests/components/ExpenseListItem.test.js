import React from 'react'
import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'
import ExpenseListItem from '../../components/ExpenseListItem'
import expenses from '../fixtures/expenses'

test('Should render Expense List Item with expense', () => {
    const expense = expenses[0]
    const wrapper = shallow(<ExpenseListItem key={expense.id} {...expense}/>)
    expect(toJSON(wrapper)).toMatchSnapshot()
})