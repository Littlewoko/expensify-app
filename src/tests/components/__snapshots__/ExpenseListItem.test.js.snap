// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Expense List Item 1`] = `
<div>
  <Link
    replace={false}
    to="/edit/1"
  >
    <h3>
      coffee
    </h3>
  </Link>
  <p>
    £
    2.5
     - 
    0
  </p>
</div>
`;

exports[`Should render Expense List Item with expense 1`] = `
<div>
  <Link
    replace={false}
    to="/edit/1"
  >
    <h3>
      coffee
    </h3>
  </Link>
  <p>
    £
    2.5
     - 
    0
  </p>
</div>
`;
